src/Admin/AdminExtensionInterface.php

```diff
@@ -32,6 +32,7 @@
  * @method void  configureDefaultFilterValues(AdminInterface $admin, array &$filterValues)
  * @method void  configureDefaultSortValues(AdminInterface $admin, array &$sortValues)
  * @method void  configureFormOptions(AdminInterface $admin, array &$formOptions)
+ * @method array configurePersistentParameters(AdminInterface $admin, array $parameters)
```

Comment:
This causes some deprecation warnings, because the `AbstractAdminExtension` does not provide this method @VincentLanglet